src/libs/actions/Report.ts

```diff
@@ -2449,6 +2456,15 @@ function leaveGroupChat(reportID: string) {
             value: null,
         },
     ];
+    // Clean up any quick actions for the report we're leaving from
+    if (quickAction?.chatReportID?.toString() === reportID) {
```

Comment:
Even though `quickAction.chatReportID` is defined as a string, I've noticed that it was actually stored as an int in Onyx. I'm not sure why, since every flow that I traced had it as a string before saving it to Onyx.

Reply:
That's strange. For me, `chatReportID` is stored as a `string` in onyx. I also do not see usage of `int` in any flow. Further, I also tried relogin and tried various flows and it still stores as `string`. Looks good to me though.